src/server/auth/cmds/cmds.go

```diff
@@ -438,6 +438,35 @@ func GetRobotTokenCmd() *cobra.Command {
 	return cmdutil.CreateAlias(getAuthToken, "auth get-robot-token")
 }
 
+func GetGroupsCmd() *cobra.Command {
+	var enterprise bool
+	getGroups := &cobra.Command{
+		Use:   "{{alias}} [username]",
+		Short: "Get an auth token for the user with the specified name.",
+		Long:  "Get an auth token for the user with the specified name.",
+		Run: cmdutil.RunBoundedArgs(0, 1, func(args []string) error {
+			c, err := newClient(enterprise)
+			if err != nil {
+				return errors.Wrapf(err, "could not connect")
+			}
+			defer c.Close()
+
+			req := &auth.GetGroupsRequest{}
+			if len(args) == 1 {
+				req.Username = args[0]
+			}
+			resp, err := c.GetGroups(c.Ctx(), req)
+			if err != nil {
+				return grpcutil.ScrubGRPC(err)
+			}
+			fmt.Println(strings.Join(resp.Groups, "\n"))
+			return nil
+		}),
+	}
+	getGroups.PersistentFlags().BoolVar(&enterprise, "enterprise", false, "Get group membership info from the enterprise server")
```

Comment:
Just checking my mental model: this is for users who want to change the IDP configs, and want to make sure that the enterprise server will authorize them to do so (because a group that they're a member of has a ClusterRole, and the group membership must therefore be stored on the enterprise sierver for the user to assume that ClusterRole and modify the IDP config)?

Reply:
Yeah, basically - the enterprise server has it's own auth server that users have to log in to, and that syncs group membership the same as the pachd. That group membership can be used in role bindings to support updating the enterprise license or configuring IDPs.